rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/RMRest.java

```diff
@@ -254,19 +254,26 @@ public RMState getState(@HeaderParam("sessionid") String sessionId) throws NotCo
     }
 
     /**
-     * Returns the initial state of the resource manager
+     * Returns difference between current state of the resource manager
```

Comment:
I am confused. Isn't it supposed to be part of the incremental RM updates PR?

Reply:
I made a mistake. I will rebase this PR